src/lib.rs

```diff
@@ -16,7 +16,7 @@ use bevy::{
     prelude::*,
     render::{
         camera::Camera,
-        mesh::{Indices, VertexAttribute, VertexAttributeValues},
```

Comment:
`VertexAttribute` no longer exists, it's been replaced with a Map and some const strings on `Mesh`